hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -90,7 +90,7 @@ def pulp_present?
   end
```

Comment:
This and `pulp_enabled?` can probably be removed now? They don't work anymore.

Reply:
Dropped alongside the hooks that relied on them